src/openms/include/OpenMS/ANALYSIS/TOPDOWN/PeakGroup.h

```diff
@@ -103,9 +103,10 @@ namespace OpenMS
            @brief Update isotope cosine sore and qscore. Mono mass is also updated one last time. SNR, per charge SNR, and avg errors are updated here.
            @param avg precalculated averagine
            @param min_cos the peak groups with cosine score less than this will have Qscore 0.
+           @param allowed_isotope_error this set the allowed isotope error in dummy mass generation.
```

Comment:
Can you elaborate a little? What unit is this error? Abosulte or relative?

Reply:
Oh. That is the number.. Like one isotope or two. I will add comment

Reply:
Ah yes, that would be good. Not sure how to name it, "isotope number" might be ok, yes.